hera_notebook_templates/tests/test_version.py

```diff
@@ -1,130 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Why delete this file?

Reply:
It was testing non-existent functionality (we removed that way of versioning in a previous PR)